src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/Breakpoint.css

```diff
@@ -54,6 +54,16 @@
   border-top: 1px solid var(--theme-splitter-color);
 }
 
+.disabled {
+  background: repeating-linear-gradient(
+    45deg,
+    var(--theme-base-95),
+    var(--theme-base-95) 5px,
+    var(--theme-base-100) 5px,
+    var(--theme-base-100) 10px
```

Comment:
This should probably use the same CSS variables as `--point-panel-background-color` and the new one, so they stay in sync?

Reply:
You'd think so, but this is on the sidebar, with a smaller area. It didn't look right using an identical treatment, so the color was lightened slightly for readability.